### PR TITLE
Pinned countries

### PR DESCRIPTION
**Pull request by dshoreman**
Sorts the countries into alphabetical order, and adds a pinning feature that lets users pin certain countries to the top of the list.

### CODE REVIEW

File: updates/create_countries_table.php [hunk 1]
```diff
@@ -15,14 +14,19 @@ public function up()
...
             return;
         }
 
-        Schema::create('rainlab_location_countries', function($table)
-        {
+        Schema::create('rainlab_location_countries', function($table) {
             $table->engine = 'InnoDB';
             $table->increments('id');
             $table->boolean('is_enabled')->default(false);
+            $table->boolean('is_pinned')->default(false);
```
Comment (daftspunk) [inline updates/create_countries_table.php]: Remove this

File: updates/create_countries_table.php [hunk 2]
```diff
@@ -15,14 +14,19 @@ public function up()
          */
         if (Schema::hasTable('rainlab_user_countries')) {
             Schema::rename('rainlab_user_countries', 'rainlab_location_countries');
+
+            Schema::table('rainlab_location_countries', function($table) {
```
Comment (daftspunk) [inline updates/create_countries_table.php]: Move this code block to a new migration file and version

**Comment by daftspunk** [comment]
This needs a new migration file for existing sites to add `is_pinned` to the database, otherwise the `orderBy` statement will fail.

**Comment by daftspunk** [comment]
Call the migration file `add_country_pinned_flag.php`

Version notes
1.0.x: Add pinned flag to the countries table to make a country appear at the top of the list.

**Comment by dshoreman** [comment]
Good call on the migration, totally skimmed over it. I'll get an update sorted in a bit

**Comment by dshoreman** [comment]
Something that crossed my mind while updating bits was `is_pinned` being added later affects the seed, too. The field won't be there when the seed runs, meaning the queries would all fail.

To get around that I've also moved the setting of `is_pinned`  on AU, GB, CA and US to a new seed.

Push incoming...

**Comment by daftspunk** [comment]
Merged in 703ce088f3e7a4604ef0da23fefa6a0021efbe81

**Pull request closed without merging**